lib/LedgerSMB/Request/Error.pm

```diff
@@ -32,11 +32,11 @@ String to send as error message
 
 =cut
 
-has msg => (is => 'ro', isa => 'Str', required => 1);
+has msg => (is => 'ro', isa => 'Object', required => 1);
```

Comment:
Why did you have to change this to an Object? I can't see why $msg should be an object in the code above: if it's an instance of LedgerSMB::Request::Error, the constructor isn't called for another wrapping.